lib/cloud_controller/diego/docker/staging_completion_handler.rb

```diff
@@ -12,8 +12,9 @@ def initialize(runners)
         def self.success_parser
           @staging_response_schema ||= Membrane::SchemaParser.parse do
             {
-              execution_metadata: String,
-              detected_start_command: Hash,
+              :execution_metadata => String,
```

Comment:
This change doesn't appear to be necessary.  The code that was here before the PR uses the new hash syntax. Is there a reason this was done or can you put it back to how it was?